tests/sentry/api/endpoints/test_group_notes.py

```diff
@@ -261,3 +262,70 @@ def test_with_group_link(self):
                 assert activity.user_id == self.user.id
                 assert activity.group == group
                 assert activity.data == {"text": comment, "external_id": "123456789"}
+
+    @with_feature("organizations:team-workflow-notifications")
+    def test_team_mentions_v2(self):
```

Comment:
Maybe something like `team_mention` vs `team_users_mention` or smth like that ? eventually we want to move over to having all teams assigned as a team , so this would be temporary anyways